src/Components/Carousel/Carousel.js

```diff
@@ -1,78 +1,133 @@
-import React, {useEffect, useState} from 'react';
-import './Carousel.scss';
-
-import useFetch from '../useFetch/useFetch';
+import React, { useEffect, useState } from "react";
+import "./Carousel.scss";
+import useFetch from "../useFetch/useFetch";
+import Modal from "../Modal/Modal";
+import { v4 as uuidv4 } from 'uuid'
 
 const Carousel = () => {
-   
-    const {data} = useFetch()
-    console.log(data);
-
-    const [offset, setOffset] = useState(0)
-    const [width, setWidth] = useState(window.innerWidth)
-    
-    let count = 4;
-
-    useEffect(() => {
-        const listener = () => {
-            setWidth(window.innerWidth)
-        }
-        window.addEventListener("resize", listener)
-        
-        return () => {
-        window.removeEventListener("resize", listener)
-        }
-        
-    },[])
-
-    console.log(width)
-
-    if (width < 900 ) {
-        count = 2
-    } 
-    if (width < 600) {
-        count = 1
-     }
-
-    const goBack = () => {
-        setOffset(offset - count)
-        }
-    
-
-    const goForward = () => {
-        setOffset(offset + count)
+  const { data, numberReq } = useFetch();
+
+  const [offset, setOffset] = useState(0);
+  const [width, setWidth] = useState(window.innerWidth);
+  const [activePage, setActivePage] = useState(0);
+
+  const [modal, setModal] = useState(false);
+  const [text, setText] = useState(null);
+
+  const showModal = (tt) => {
```

Comment:
For parameter names, it is not obligatory, but it's very useful if it's more descriptive.